### PR TITLE
Fix lint errors blocking rebase of release to master branch

### PR DESCRIPTION
**Pull request by gargnitingoogle**
### Description
This fixes all the [lint errors](https://github.com/GoogleCloudPlatform/gcsfuse/actions/runs/6310787426/job/17164364560?pr=1400) blocking the rebase of release to master branch (https://github.com/GoogleCloudPlatform/gcsfuse/pull/1400)

### Link to the issue in case of a bug fix.
NA

### Testing details
1. Manual - Did manual testing locally for following operations: list, stat, read, write, delete-in-mount-directory, copy-in-mount-directory, delete-from-gcs, copy-in-gcs .
2. Unit tests - Ran locally and all passed.
3. Integration tests - Will run as part of presubmit using the presubmit integration-tests label.

### CODE REVIEW

**Comment by gargnitingoogle** [comment]
This PR will be `squash and rebase`'ed into master branch on approval.

File: internal/contentcache/contentcache.go [hunk 1]
```diff
@@ -152,11 +151,19 @@ func (c *ContentCache) RecoverCache() error {
...
+	dirEntries, err := os.ReadDir(c.tempDir)
 	if err != nil {
 		// if we fail to read the specified directory, log and return error
 		return fmt.Errorf("recover cache: %w", err)
 	}
+	files := make([]os.FileInfo, len(dirEntries))
+	for i, dirEntry := range dirEntries {
+		files[i], err = dirEntry.Info()
+		if err != nil {
+			// if we fail to read the specified directory, log and return error
```
Comment (vadlakondaswetha) [inline internal/contentcache/contentcache.go:163]: you need to update the comment here. We failed to read the specific directory entry.

File: internal/gcsx/integration_test.go [hunk 2]
```diff
@@ -137,11 +136,6 @@ func (t *IntegrationTest) objectGeneration(name string) (gen int64) {
 		panic(err)
 	}
 
-	// Check the result.
-	if o.Generation > math.MaxInt64 {
-		panic(fmt.Sprintf("Out of range: %v", o.Generation))
```
Comment (vadlakondaswetha) [inline internal/gcsx/integration_test.go:142]: why is this removed?

Comment (gargnitingoogle) [reply internal/gcsx/integration_test.go:142]: This is an pointless check, it's comparing an int64 value `>` MaxInt64, which is always false by definition.

File: internal/contentcache/contentcache.go (hunk 1 above)
Comment (gargnitingoogle) [reply internal/contentcache/contentcache.go]: Done.

File: internal/contentcache/contentcache.go [hunk 3]
```diff
@@ -152,11 +151,21 @@ func (c *ContentCache) RecoverCache() error {
...
-		// if we fail to read the specified directory, log and return error
+		// We failed to get the list of sub-directories
+		// in the temp directory, log and return error.
 		return fmt.Errorf("recover cache: %w", err)
 	}
+	files := make([]os.FileInfo, len(dirEntries))
+	for i, dirEntry := range dirEntries {
+		files[i], err = dirEntry.Info()
+		if err != nil {
+			// We failed to read the specified directory,
```
Comment (vadlakondaswetha) [inline internal/contentcache/contentcache.go:164]: we are reading files inside the directory. either call it directory entry or file. read the specified directory is wrong word here.

Comment (gargnitingoogle) [reply internal/contentcache/contentcache.go]: @vadlakondaswetha  Done. PTAL.

Comment (gargnitingoogle) [reply internal/contentcache/contentcache.go]: @vadlakondaswetha ^

**Review by vadlakondaswetha — APPROVED** [review]
